client/state/selectors/get-blog-stickers.js

```diff
@@ -8,5 +10,5 @@ import { get } from 'lodash';
  * @returns {Array} Blog stickers
  */
 export default function getBlogStickers( state, blogId ) {
-	return get( state.sites.blogStickers.items, blogId, null );
+	return get( state.blogStickers.items, blogId, null );
```

Comment:
nit: could we use the opportunity to remove that `get` call as well?

Reply:
Good call. Done in 49036f56d9a064c9c53857dc726a70a4187bf2c3.